Lib/test/test_fcntl.py

```diff
@@ -129,9 +129,9 @@ def test_fcntl_bad_file_overflow(self):
             fcntl.fcntl(BadFile(INT_MIN - 1), fcntl.F_SETFL, os.O_NONBLOCK)
 
     @unittest.skipIf(
-        platform.machine().startswith(("arm", "aarch"))
-        and platform.system() in ("Linux", "Android"),
-        "ARM Linux returns EINVAL for F_NOTIFY DN_MULTISHOT")
+        (platform.machine().startswith("arm") and platform.system() == "Linux")
+        or platform.system() == "Android",
+        "this platform returns EINVAL for F_NOTIFY DN_MULTISHOT")
     def test_fcntl_64_bit(self):
```

Comment:
It turns out this test works fine on Linux aarch64, but it doesn't work on Android on any architecture. So I've reverted the Linux part of the condition to the state it was in before #115918.